### DISCUSSION TITLE
Save live timing windows (including sub-windows) in saved layout

### DISCUSSION THREAD

**Discussion by martijnvankekem** [f1multiviewer/issue-tracker#439] [Feature requests] [answered]
When opening a saved layout, all players are correctly positioned, but I need to manually open the live timing windows, together with the AI radio transcriptions, and move them to the correct place.

It'd be great if this could be saved together with the rest of the player windows.

**Comment by f1multiviewer** [comment, marked as answer]
Duplicate of #321
